### PR TITLE
refactor: [M3-6522] - BetaChip V7 story migration and cleanup

### PR DESCRIPTION
**Pull request by abailly-akamai**
## Description 📝
Small PR to migrate the BetaChip component story to a V7 (`.tsx`) story 

## Changes  🔄
- Clean up `<BetaChip />` (props, export, styling)
- New v7 `.tsx` story to replace the `.mdx` one
- Update storybook to `7.5.2`

## Preview 📷
This PR should not introduce any visual change

### Prerequisites
- pull code locally
- run `yarn && yarn storybook`

### Verification steps 
- navigate to http://localhost:6006/?path=/docs/components-chip-betachip--documentation
- confirm story loads properly
- start cloud manager and confirm there's no regression for the beta chip

## As an Author I have considered 🤔

*Check all that apply*

- [x] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [x] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/package.json [hunk 1]
```diff
@@ -202,8 +202,8 @@
     "redux-mock-store": "^1.5.3",
     "reselect-tools": "^0.0.7",
     "serve": "^14.0.1",
-    "storybook": "~7.0.24",
-    "storybook-dark-mode": "^3.0.0",
+    "storybook": "~7.5.2",
+    "storybook-dark-mode": "^3.0.1",
```
Comment (abailly-akamai) [inline packages/manager/package.json:206]: The update was all run using `npx storybook@latest upgrade`

should feature latest updates for the package

File: packages/manager/src/components/BetaChip/BetaChip.tsx [hunk 2]
```diff
@@ -15,38 +14,45 @@ interface BetaChipProps
     | 'label'
     | 'onDelete'
     | 'outlineColor'
+    | 'size'
```
Comment (abailly-akamai) [inline packages/manager/src/components/BetaChip/BetaChip.tsx:17]: we don't really want the size here and it wasn't being used

File: packages/manager/src/components/BetaChip/BetaChip.tsx [hunk 3]
```diff
@@ -15,38 +14,45 @@ interface BetaChipProps
...
     | 'variant'
   > {
-  className?: string;
```
Comment (abailly-akamai) [inline packages/manager/src/components/BetaChip/BetaChip.tsx:20]: className is already available as part of `Chip` so no need to re-declare it here

**Review by cpathipa — APPROVED** [review]
- [x] Confirming the story loads properly
- [x] confirming  there's no regression for the beta chip.
image
image
image

**Review by mjac0bs — APPROVED** [review]
Thanks for the clean up! The Storybook story looks good and beta chips in Cloud look good in both light/dark modes, no console errors. 🚢

**Pull request merged**